tests/cypress/integration/actions_tasks/case_120_change_completed_jobs_task.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2021-2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Task change completed jobs.', () => {
+    const caseId = '120';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const attrName = `Attr for ${labelName}`;
+    const textDefaultValue = 'Some default value for type Text';
+    const imagesCount = 15;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const advancedConfigurationParams = {
+        multiJobs: true,
+        segmentSize: 5,
+    };
+
+    before(() => {
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.visit('auth/login');
+        cy.login();
+        cy.createAnnotationTask(
+            taskName,
+            labelName,
+            attrName,
+            textDefaultValue,
+            archiveName,
+            false,
+            advancedConfigurationParams,
+        );
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.deleteTask(taskName);
+    });
+
+    function checkCompletedJobs(count, total) {
+        cy.get('.cvat-tasks-list-item')
+            .within(() => {
+                cy.get('div').eq(3).should('contain.text', `${count} of ${total} jobs`);
+            });
+    }
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Changed numbers of completed job.', () => {
+            checkCompletedJobs(0, 3);
+            cy.openTask(taskName);
+            cy.setJobStage(0, 'acceptance');
+            cy.goToTaskList();
+            checkCompletedJobs(1, 3);
+            cy.openTask(taskName);
+            cy.setJobStage(1, 'acceptance');
+            cy.goToTaskList();
+            checkCompletedJobs(2, 3);
+            cy.openTask(taskName);
+            cy.setJobStage(2, 'acceptance');
```

Comment:
To simplify I would suggest this:

```suggestion
            cy.setJobStage(0, 'acceptance');
            cy.setJobStage(1, 'acceptance');
            cy.setJobStage(2, 'acceptance');
```

Reply:
Made the test more universal.

I still think that in UI tests you should go through the interface and check the status of elements. In this test it doesn't take much time.